config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
   root "articles#index"
 
-  get "/articles", to: "articles#index"
+  resources :articles
```

Comment:
What does resources do?

Reply:
The resource maps all conventional paths to a collection of resources that work together to perform CRUD operations, in this case, the articles.